### ISSUE TITLE
Effects are not applying when keyboard is hide

### ISSUE THREAD

**Issue by IntuzMayurS** [niendo1/richeditor-android#4]
Scenario for reproducing:

Run sample app
Focus on editor, type  any content
Now, I have Open Color picker dialog. That  i am hide the keyboard and choose the color. 
This color is not set when i am writing into editor

**Comment by niendo1** [comment]
I can confirm that...

**Comment by niendo1** [comment]
can you try webView.requestFocusFromTouch() before setTextColor(). Sometimes it brings back the keyboard, but not always..

**Comment by niendo1** [comment]
if you use the latest commit: 
   implementation 'com.github.niendo1:richeditor-android:a82b781ac1'

something like this helps:
```
    @Override
    public boolean onResult(@NonNull String dialogTag, int which, @NonNull Bundle extras) {
        if (which == BUTTON_POSITIVE) {
            switch (dialogTag) {
                case DLG_HTML_TXT_COLOR:
                    lastTxtColor = extras.getInt(SimpleColorDialog.COLOR);
                    editText.requestFocusFromTouch();
                    editText.setOnJSDataListener(value -> {
                        editText.requestFocusFromTouch();
                        editText.setTextColor(lastTxtColor);
                    });
                    editText.getSelectedHtml(); // just a dirty Hack!
                    return true;
```

It seems, when the dialog is calling onResult, the dialog still prevents the focus to the editor.
But with this hack its working on ImapNotes.

New version will be released in the next days/week.

**Comment by IntuzMayurS** [comment]
@niendo1 Thank you for the support. I also resolved issue using the below method override into the RichEditor class. 

I just give the focus again to editor.
  @Override
  public void onWindowFocusChanged(boolean hasWindowFocus) {
    super.onWindowFocusChanged(hasWindowFocus);
    if (hasWindowFocus) {
      requestFocus();
      post(() -> {
        InputMethodManager imm = getContext().getSystemService(InputMethodManager.class);
        imm.showSoftInput(this, 0);
      });
    }
  }

I think this is the correct way to get again focus get into the editor.

**Comment by niendo1** [comment]
@IntuzMayurS 

Cool!
I added the fix in commit a32bc31cc1. And it works so much better!
